sharding/chainParametersHolder.go

```diff
@@ -39,24 +41,19 @@ func NewChainParametersHolder(args ArgsChainParametersHolder) (*chainParametersH
 		return nil, ErrMissingConfigurationForEpochZero
 	}
 
-	currentParams, err := getMatchingChainParametersUnprotected(args.EpochNotifier.CurrentEpoch(), args.ChainParameters)
-	if err != nil {
-		return nil, err
-	}
-
 	paramsHolder := &chainParametersHolder{
-		currentChainParameters: currentParams,
+		currentChainParameters: earliestChainParams, // will be updated on the epoch notifier handlers
```

Comment:
yup, that is the behavior right now. I do not see a good reason why we should change this in the future.